pkg/report/writer.go

```diff
@@ -83,7 +83,7 @@ func (tw TableWriter) Write(results Results) error {
 // TODO: refactror and fix cyclometic complexity
 func (tw TableWriter) write(result Result) {
 	table := tablewriter.NewWriter(tw.Output)
-	header := []string{"Library", "Vulnerability ID", "Severity", "Installed Version", "Fixed Version"}
+	header := []string{"Library", "Vulnerability ID", "Severity", "Installed Version", "Fixed Version", "AVD"}
```

Comment:
```suggestion
	header := []string{"Library", "Vulnerability ID", "Severity", "Installed Version", "Fixed Version", "URL"}
```

`AVD` is not a word users will be familiar with. We probably should use a more generic word such as `URL`, `Reference`, and `Link`. Also, we might be able to display a link other than AVD in the future.

Reply:
OK updated  https://github.com/aquasecurity/trivy/pull/709/commits/d1b3bcb5f5760c0b006b40bc05a687738e6c6936